actions/image_partition_action.go

```diff
@@ -75,6 +75,14 @@ labels (i.e. `raw`)
 - partlabel -- label for the partition in the GPT partition table. Defaults
 to the `name` property of the partition. May only be used for GPT partitions.
 
+- parttype -- set the partition type in the partition table. The string should
+be in a hexadecimal format (2-characters) for msdos partition tables and GUID format
+(36-characters) for GPT partition tables. For instance, "82" for msdos sets the
+partition type to Linux Swap. Whereas "0657fd6d-a4ab-43c4-84e5-0933c84b4f4f" for
+GPT sets the partition type to Linux Swap.
+For msdos partition types hex codes see: https://en.wikipedia.org/wiki/Partition_type
+For gpt partition type GUIDs see: https://systemd.io/DISCOVERABLE_PARTITIONS/
```

Comment:
not a blocker for this PR; but would be nice to add a nicer way for the user to do this rather then putting in magic numbers

Reply:
yeah, the plan is to expose the GUIDs to the recipes in a follow-up MR